modules/apigee_edge_apidocs/apigee_edge_apidocs.info.yml

```diff
@@ -0,0 +1,8 @@
+name: 'Apigee Edge: API Docs'
+type: module
+description: 'Display OpenAPI documentation of your APIs to your developers.'
+core: 8.x
+package: 'Apigee (Experimental)'
```

Comment:
Suggestion: You can provide a ```configure: apidocs.settings```key which adds a nice little configure icon on the admin -> extend page.